src/cluster/src/helm.rs

```diff
@@ -101,6 +101,23 @@ impl HelmClient {
         })
     }
 
+    /// Get all the available versions
+    pub fn get_versions(&self, chart: &str) -> Result<Vec<Chart>, IoError> {
```

Comment:
I think it'd be good to expose `HelmChart` as public, so we might want to consider making a `HelmError` type to go along with it, especially if we consider breaking it into its own crate one day (which I think could be a good possibility).

Reply:
Yes, there is separate issue for moving HelmClient as separate crate